docs/source/api/apollo-server.md

```diff
@@ -2,12 +2,149 @@
 title: apollo-server
 ---
 
-> WIP / TODO.
+This API reference documents the exports from the `apollo-server`, and `apollo-server/{variant}` packages (e.g. `apollo-server/express`).  Both the root module and the module sub-paths export the same functionality.
 
-## `ApolloServer` constructor
+## `ApolloServer`
+
+The `ApolloServer` class is used to create an instance of Apollo Server.  An instance of Apollo Server will only be available when `listen` is called.
+
+If using Apollo Server as middleware in an existing project, it is also necessary to call `applyMiddleware`.
+
+### `new ApolloServer(options)`: <`ApolloServer`>
```

Comment:
👍 I clearly hadn't considered the way this would appear in the sidebar menu (nor did I look at the Netlify preview).

I'm going to try going with `ApolloServer` and dropping this into a `constructor(options)` within it, then put "Usage" under that.